ntoskrnl/ex/time.c

```diff
@@ -401,6 +452,8 @@ NtSetSystemTime(IN PLARGE_INTEGER SystemTime,
     TIME_FIELDS TimeFields;
     KPROCESSOR_MODE PreviousMode = ExGetPreviousMode();
     NTSTATUS Status = STATUS_SUCCESS;
+    RTL_TIME_ZONE_INFORMATION TimeZoneInformation = { 0 };
+    ULONG ExpTimeZoneIdSave;
```

Comment:
Please remove the `Exp` prefix, that looks like it would be a global variable.

Reply:
Removed the ```Exp``` variable prefix. Thanks.